engine/src/main/java/org/entando/entando/web/common/EntandoMessageCodesResolver.java

```diff
@@ -0,0 +1,64 @@
+package org.entando.entando.web.common;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.springframework.util.StringUtils;
+import org.springframework.validation.DefaultMessageCodesResolver;
+import org.springframework.validation.MessageCodeFormatter;
+
+public class EntandoMessageCodesResolver extends DefaultMessageCodesResolver {
```

Comment:
Please add a high level description of this in a comment. This is pretty invisible functionality so we need to make sure to describe what's going on here.